### ISSUE TITLE
Incorrect characters being displayed with `addText()`

### ISSUE THREAD

**Issue by transcental** [hackclub/sprig#1758]
**Describe the bug**
When using `addText()` with the pipe operator, `|`, `ö` is displayed instead.

Whilst I discovered this with `|`, it's also an issue for other symbols such as `~`, `±` and `§`. These are shown respectively in the bottom screenshot.

**To Reproduce**
1. Make a Sprig project.
2. Use `addText()` with `|`
3. Run the game
4. See `ö` displayed

**Expected behavior**
The correct character to be displayed.

**Actual behavior**  
A different character displayed.

**Screenshots**
![image](https://github.com/hackclub/sprig/assets/83948303/fdcfaab5-4b2d-4fb8-8f27-6de42e604084)
*`|` being displayed*
<img width="214" alt="image" src="https://github.com/hackclub/sprig/assets/83948303/32bc1e16-4c1a-4a95-a1e2-d50a6529ee5e">
*`~`, `±` and `§` being displayed*

**Comment by snoglobe** [comment]
is it just my phone or are the provided screenshots corrupted? ![image](https://github.com/hackclub/sprig/assets/25540164/650879e6-ade0-4b9f-978b-223b5e901976)![image](https://github.com/hackclub/sprig/assets/25540164/f8e214d2-ceb1-40b8-928a-8ca6799be57f)

**Comment by lucas11222** [comment]
yea its your phone

**Comment by transcental** [comment]
No, they're not corrupted - that's the bug that this report is about
